tests/Integration/Console/Scheduling/ScheduleGroupTest.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Illuminate\Tests\Integration\Console\Scheduling;
+
+use Illuminate\Console\Scheduling\Schedule as ScheduleClass;
+use Illuminate\Support\Facades\Schedule;
+use Orchestra\Testbench\TestCase;
+use PHPUnit\Framework\Attributes\DataProvider;
+
+final class ScheduleGroupTest extends TestCase
```

Comment:
Final classes aren't used for framework test cases.

```suggestion
class ScheduleGroupTest extends TestCase
```